dashboard/app/models/user_script.rb

```diff
@@ -11,6 +11,7 @@
 #  last_progress_at :datetime
 #  created_at       :datetime
 #  updated_at       :datetime
+#  properties       :text(65535)
```

Comment:
Is this still the preferred way to store serialized properties or have we started moving over to the JSON data type?

Reply:
We haven't used it anywhere yet. Do we support it now? I thought we still needed an upgrade, but maybe that's done. @wjordan?

If it's supported I'd vote to start using it.

Reply:
per discussion in `#infrastructure`, we'll leave the as text and can still add a field-specific index on it later if needed.